src/tasks/eth/022-U16a-opcm-upgrade-v410-base/README.md

```diff
@@ -6,10 +6,128 @@ Status: [DRAFT, NOT READY TO SIGN]()
 
 Upgrade Base Mainnet to U16a. More context on U16a can be found in the Optimism docs [here](https://docs.optimism.io/notices/upgrade-16a).
 
-## Simulation, Signing & Execution
+## Step 1. Install Homebrew
```

Comment:
Why aren't you instructing users to install `mise` then have them simply run `mise install`? See installation steps here: https://github.com/ethereum-optimism/superchain-ops?tab=readme-ov-file#quick-start